spyderlib/plugins/shortcuts.py

```diff
@@ -159,17 +63,12 @@ def data(self, index, role=Qt.DisplayRole):
         column = index.column()
         if role == Qt.DisplayRole:
             if column == CONTEXT:
-                return to_qvariant(shortcut.context)
+                return to_qvariant(_(shortcut.context.capitalize()))

```

Comment:
Is `_` not just a function call, where we pass in the *result*, which is the computed string?